content/blog/storage-management-with-redfish.md

```diff
@@ -33,18 +33,26 @@ The reader should have the knowledge of HTTP [request methods](https://en.wikipe
 
 Moreover, it is assumed that the reader knows how to manage Redfish sessions as described in the [Redfish API Reference documentation](https://hewlettpackard.github.io/ilo-rest-api-docs/ilo5/#authentication-and-sessions). More hints for managing iLO sessions with Redfish can be found in this [article](/blog/managing-ilo-sessions-with-redfish).
 
-## Storage information locations
+## Storage data models
 
-The Redfish standard defines a `storage` subsystem as part of the `ComputerSystem` type under the `/redfish/v1/Systems/{item}/` location.
+The Redfish standard defines a `storage` data model as part of the `ComputerSystem` resource type under the `/redfish/v1/Systems/{item}/` URI. With the implementation of new standards like the Platform Level Data Model for Redfish Device Enablement ([PLDM for RDE](https://developer.hpe.com/blog/overview-of-the-platform-level-data-model-for-redfish%C2%AE-device-enablement-standard/)), this model is fully operational in terms of read, write and event operations against modern external provider devices. Starting at Gen10 and Gen10 Plus (firmware 2.30+), HPE servers can take advantage out of it. Note that it is the only storage data model implemented in Gen11 servers and beyond.
 
-However, HPE implements the description and the configuration of its Smart Storage devices (i.e. SmartArray controllers) respectively in the `SmartStorage` and `SmartStorageConfigX` (**X** is being an ID number) entry points of the same `ComputerSystem` type. When multiple controllers are present in a system,
+HPE initially developed the `SmartStorage` Redfish OEM data model for HPE ProLiant DL580 Gen8 servers, before any Redfish specification. This model supports inventory (GET) and monitoring (Events) features only.
+
+In HPE ProLiant Gen10, the `SmartStorageConfig` resource was added to support configuration. This OEM model uses a proprietary API that only supports the SmartRAID (SR) line of HPE storage controllers. 
+
+> **Note**: The HPE OEM `SmartStorageConfig` data model is removed in HPE Gen11 servers. 
+
+> **Note**: This article focuses only on the HPE OEM `SmartStorageConfig` data model.
+
+As said earlier, HPE Gen10 and Gen10 Plus servers implements the description and the configuration of its Smart Storage SR devices respectively in the `SmartStorage` and `SmartStorageConfigX` (**X** is being an ID number) entry points of the same `ComputerSystem` type. When multiple controllers are present in a system,
```

Comment:
Change "said" to "stated".
"Implements" should be "implement" because "servers" is plural.
And, since you've presented them as multiples, "...configuration of its Smart Storage" should be "configuration of their Smart Storage"